src/components/AutoComplete/AutoComplete.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <Combobox as="div" v-model="displayValue" :disabled="disabled">
+    <div :class="['relative', disabledClass]">
+      <div
+        class="
+          inline-flex
+          w-full
+          cursor-default
+          overflow-hidden
+          border
+          text-left
+          focus:outline-none
+          focus:ring-1
+          focus:border-indigo-light
+          focus:ring-indigo-light
+        "
+      >
+        <ComboboxInput
+          @change="searchValue = $event.target.value"
+          @click="$event.target.select()"
+          autocomplete="off"
+          class="w-full"
+          :class="[ sizeClass, bgColor]"
+        />
+      </div>
+      <ComboboxOptions
+        :class="[
+          'absolute z-10 mt-1 w-full bg-white shadow-lg py-1 text-base ring-1 ring-black ring-opacity-5 overflow-auto focus:outline-none sm:text-sm',
+        ]"
```

Comment:
minor: can this just be a string?
```suggestion
        class="absolute z-10 mt-1 w-full bg-white shadow-lg py-1 text-base ring-1 ring-black ring-opacity-5 overflow-auto focus:outline-none sm:text-sm"
```
You could always multiline it if it's too long but best to stick to using strings for static classes